app/src/main/res/values/strings.xml

```diff
@@ -28,6 +28,7 @@
     <string name="what_time">What time works?</string>
     <string name="pick_date">Pick a date...</string>
     <string name="at">at</string>
+    <string name="no_search_results">No Search Results!\nTry another search</string>
```

Comment:
Can you use the same message that is used in EventBrite. Their message is a little different

Reply:
updated message :+1: